### PR TITLE
Bastok 8-2 fix

### PR DESCRIPTION
**Pull request by Turttle**
Now all mission variable start at 0. The change is required for player to progress.

### CODE REVIEW

File: scripts/zones/Zeruhn_Mines/npcs/Drake_Fang.lua [hunk 1]
```diff
@@ -25,18 +25,21 @@ end;
...
+	if(currentMission == RETURN_OF_THE_TALEKEEPER and MissionStatus >= 1) then
+		if(MissionStatus == 1) then
 			player:startEvent(0x00ca);
 		else
 			player:startEvent(0x00cb);
 		end
-		
-	elseif(player:getCurrentMission(BASTOK) == ENTER_THE_TALEKEEPER and player:getVar("MissionStatus") == 1) then
+	-- Bastok 8-1	
+	elseif(currentMission == ENTER_THE_TALEKEEPER and MissionStatus == 0) then
```
Comment (whasf) [inline scripts/zones/Zeruhn_Mines/npcs/Drake_Fang.lua]: Won't MissionStatus be null if the variable doesn't exist?

Comment (teschnei) [inline scripts/zones/Zeruhn_Mines/npcs/Drake_Fang.lua]: Vars default to 0 if they aren't in the database

File: scripts/zones/Zeruhn_Mines/npcs/Drake_Fang.lua [hunk 2]
```diff
@@ -25,18 +25,21 @@ end;
...
 		end
-		
-	elseif(player:getCurrentMission(BASTOK) == ENTER_THE_TALEKEEPER and player:getVar("MissionStatus") == 1) then
+	-- Bastok 8-2	
+	elseif(currentMission == ENTER_THE_TALEKEEPER and MissionStatus == 0) then
 		player:startEvent(0x00ca);
-	elseif(player:getCurrentMission(BASTOK) == ENTER_THE_TALEKEEPER) then
-		if(player:getVar("MissionStatus") == 4) then
+	elseif(currentMission == ENTER_THE_TALEKEEPER) then
+		if(MissionStatus == 4) then
```
Comment (xdemolish) [inline scripts/zones/Zeruhn_Mines/npcs/Drake_Fang.lua]: why not just include the MissionStatus check in the elseif?

**Comment by xdemolish** [comment]
reopen/create a new pull request when updated

**Comment by Turttle** [comment]
ok

**Pull request closed without merging**